src/react/account/AccountUserList.tsx

```diff
@@ -126,6 +138,49 @@ const WithTooltipWhileLoading = ({
   </>
 );
 
+const DeleteUserAction = (rowValue: { userName : string} , accountName: string) => {
+  const { userName } = rowValue;
+  const IAMClient = useIAMClient();
+  const [showModal, setShowModal] = useState(false);
+  const { data: accessKeysResult } = useAwsPaginatedEntities(getUserAccessKeysQuery(userName, notFalsyTypeGuard(IAMClient)), data => data.AccessKeyMetadata);
+  const { data: listGroupsResult } = useAwsPaginatedEntities(getUserListGroupsQuery(userName, notFalsyTypeGuard(IAMClient)), data => data.Groups);
```

Comment:
Thank you for your comment. Is it mandatory to use useQuery in this case, because I tried to use it but I noticed that the delete button stayed enabled all the time?  If it should be used can you please tell me what's wrong with my query at the line 145 in AccountUserList.tsx ? Thanks!